assignment_w48.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <cstdlib>
+#include <algorithm>
+#include <fstream>
+#include <vector>
+#include <string>
+using namespace std;
+
+/*  create a deck of cards that may be sorted using either their
+value (e.g., all aces, then all twos, then all threes ...) or suite-value combination.
+Finally make it possible to shuffle the deck. */
+
+// a card must be represented as a type with a value . eg: 7 of spades will be 7S, Ace of clubs will be 1C, King of diamonds
+// would be 13D...or KD
+
+ class Card
+{
+public:
+    string faces;
```

Comment:
Bad naming: Why the name is plural? I do not agree with the data type either.

Reply:
Have to agree. This would be better as the name of the enum